lightningd/chaintopology.c

```diff
@@ -277,12 +319,17 @@ static void update_feerates(struct bitcoind *bitcoind,
 	bool changed = false;
 
 	for (size_t i = 0; i < NUM_FEERATES; i++) {
-		if (satoshi_per_kw[i] != topo->feerate[i])
+		u32 feerate = satoshi_per_kw[i];
+
+		if (feerate < feerate_floor())
+			feerate = feerate_floor();
```

Comment:
Suggest adding a `log_debug` here that we are bumping the feerate up because reasons.

Reply:
Done.